logging/access.go

```diff
@@ -0,0 +1,126 @@
+package logging
+
+import (
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"net"
+	"net/http"
+	"time"
+)
+
+const (
+	dateFormat      = "02/Jan/2006:15:04:05 -0700"
```

Comment:
Is that how go date format works? You just throw it a date and it figures it out? Weird magic. Anyway, is the separation between date and time really a `:`?